doc/3/controllers/document/m-update/index.md

```diff
@@ -0,0 +1,82 @@
+---
+code: true
+type: page
+title: mUpdate
+description: Updates multiple documents
+---
+
+# mUpdate
+
+Updates multiple documents.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, ArrayList<Object>>> mUpdate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, ArrayList<Object>>> mUpdate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents,
+      final DocumentOptions options)
```

Comment:
see https://github.com/kuzzleio/sdk-java/pull/59#discussion_r388769189

Reply:
I'm thinking that a could use the same class for `document:method` as for `document:mMethod` what do you tkink ?
A `CreateDocumentOptions` for `document:create` and `document:mCreate`,
A `UpdateDocumentOptions` for `document:update` and `document:mUpdate`
etc..

Reply:
Of course. See our documentation about optional options: https://github.com/kuzzleio/meeting-notes/blob/master/specs/2019-06-20%20Optional%20parameters%20in%20SDKs.md#options-that-persist-even-after-the-function-has-been-called

```
We have to build as many dedicated classes as necessary. Each class must have one use only: for instance, the SearchOptions class can only be used by document:search, security:searchRoles, security:searchProfiles, ... since all these functions can use the exact same options.

But we cannot add optional properties unrelated to search functions in it.
```